app/controllers/regular_transactions_controller.rb

```diff
@@ -0,0 +1,18 @@
+class RegularTransactionsController < ApplicationController
+  def create
+    if creation_service.success?
+      render_success
+    else
+      render_unprocessable(creation_service.errors)
+    end
+  end
+
+private
+
+  def creation_service
+    @creation_service ||= Creators::RegularTransactionsCreator.call(
+      assessment_id: params[:assessment_id],
+      regular_transaction_params: request.raw_post,
+    )
```

Comment:
Just wondering if we need to memo-ize this?

It looks like it's only called once per request-response cycle, so I'd be inclined not to.

Reply:
ty, this just follows the pattern of existing controllers and is quite probably redundant but would be inclined to leave unless we apply to all